src/Sensor/MemoryMeasurement.java

```diff
@@ -9,17 +9,18 @@
     MemoryMeasurement(Sigar _sigar){
 
         super(_sigar,"MemoryUsage", "Megabytes");
+        setMeasureMaxValue(calculateMaxValue());
```

Comment:
What happends if you pass "Error while calculating max value" to `setMeasureMaxValue()`?